src/app/config/selectors.js

```diff
@@ -18,8 +18,12 @@ export const getFilteredCollections = createSelector(getMappedCollections, getSe
         return name.includes(string);
     });
 });
+
 export const getWorkingOn = state => state.global.workingOn;
 export const getActive = state => state.collections.active;
 export const getCollectionsLoading = state => state.collections.isLoading;
 export const getCollectionCreating = state => state.collections.isCreating;
 export const getIsUpdatingCollection = state => state.collections.isUpdating;
+export const selectNewTeamUsersNotInTeam = state => state.newTeam.usersNotInTeam;
```

Comment:
is this needed.. again naming confusing. Worth checking  https://redux.js.org/style-guide/style-guide#organize-state-structure-based-on-data-types-not-components

Reply:
Have removed the one highlighted here (as it is unused) and renamed the two under it to be more descriptive of what they are storing (a little verbose but better to be clear and verbose than short and unclear)
`selectNewTeamAllPreviewUsers` -> `selectPreviewUsersNotInCurrentTeam` this is the list of preview users not assigned to the current team. These are the equivalent to users that used to have the 'role' `viewer` in Zebedee and who are not in the current team the user is creating (but could be in other teams)
`selectNewTeamUnsavedChanges` -> `selectUnsavedChangesOnTeamCreation`